jnipp.h

```diff
@@ -539,7 +539,7 @@ namespace jni
          */
         template <class TReturn>
         TReturn call(const char* name) const {
-            method_t method = getMethod(name, ("()" + internal::valueSig((TReturn*) nullptr)).c_str());
+            method_t method = callStaticMethod(name, ("()" + internal::valueSig((TReturn*) nullptr)).c_str());
```

Comment:
`callStaticMethod()` takes a `method_t` parameter, not a `const char*` parameter.  It also requires a type specifier for the return type.